### DISCUSSION TITLE
Export to godot project

### DISCUSSION THREAD

**Discussion by ghost** [4ian/GDevelop#5260] [Archive] [answered]
basically I started learning how to make a Godot game and gdscript but the big problem is that there is no easy way as it's not well documented so I think making an export from gdevelop to godot might be an easy way to understand it.

I don't care if it's official or third-party, I just only need for the converter to output code that is not ugly

**Comment by arthuro555** [comment]
that is not possible.

**Comment by girng** [comment]
why would you ever want to do that

**Comment by Silver-Streak** [comment, marked as answer]
Hello!

There is no relationship between Godot and GDevelop. Their code and game development methods are not compatible, so there is not a way to go from one engine to the other.

If you are wishing to learn about Godot I'd recommend reviewing their documentation or the myriad of tutorials available across the internet.
